notebooks/Binary Classification with the UCI Credit-card Default Dataset.ipynb

```diff
@@ -608,41 +627,29 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "model_sweep_dict = {\"GridSearch_{}\".format(i): sweep_preds[i] for i in range(len(sweep_preds)) if non_dominated[i]}\n",
-    "model_sweep_dict.update({\"Unmitigated\": test_preds, \"ThresholdOptimizer\": postprocess_preds})"
+    "# Compare GridSearch models with low values of equalized odds difference with the previously constructed models\n",
+    "grid_search_dict = {\"GridSearch_{}\".format(i): (sweep_preds[i], sweep_scores[i])\n",
+    "                    for i in range(len(sweep_preds))\n",
+    "                    if non_dominated[i] and equalized_odds_sweep[i]<0.1}\n",
+    "models_dict.update(grid_search_dict)\n",
+    "get_metrics_df(models_dict, Y_test, A_str_test)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Grid Search: Dashboard Demo**"
+    "## Conclusion"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We compare the `GridSearch` candidate models with the unmitigated `lightgbm` model and the `ThresholdOptimizer` model using the Fairlearn dashboard. We can select the balanced accuracy as the performance metric and examine the trade-off curve between balanced accuracy and the disparity in balanced accuracy."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "FairlearnDashboard(sensitive_features=A_str_test, sensitive_feature_names=['Sex'],\n",
-    "                   y_true=Y_test,\n",
-    "                   y_pred=model_sweep_dict)"
+    "In this notebook, we explored how a fairness-unaware gradient boosted trees model performed on the classification task in contrast to the postprocessed `ThresholdOptimizer` model and the `GridSearch` model. The `ThresholdOptimizer` greatly reduced the disparity in performance across multiple fairness metrics. However the overall error rate and AUC for the `ThresholdOptimizer` model were worse compared to the fairness-unaware model. \n",
```

Comment:
"the GridSearch model" --> in the cell above it's plural "models", so we should probably be consistent.